make/targets/openshift/operator/run-local.mk

```diff
@@ -0,0 +1,35 @@
+self_dir :=$(dir $(lastword $(MAKEFILE_LIST)))
+scripts_dir :=$(self_dir)../../../../scripts
+current_dir = $(shell pwd)
+
+define add-run-local-internal
+# Only set env vars if not already set to allow the calling shell's
+# environment to take precedence.
+ifndef RL_DEPLOYMENT_YAML
+export RL_DEPLOYMENT_YAML = $(current_dir)/$(1)
+endif
+ifndef RL_CMD_PATH
+export RL_CMD_PATH = $(current_dir)/$(2)
+endif
+ifndef RL_LOCK_CONFIGMAP
+export RL_LOCK_CONFIGMAP = $(3)
+endif
+ifndef RL_CMD
+export RL_CMD = $(4)
+endif
+ifndef RL_ARGS
+export RL_ARGS = $(5)
+endif
+ifndef RL_BUILD_FLAGS
+export RL_BUILD_FLAGS = $(6)
+endif
+run-local:
```

Comment:
if this is running telepresence, let's name it like that (`run-telepresence`), there are other ways to run locally

Reply:
Done